Nio/Authentication/LoginView.swift

```diff
@@ -111,10 +130,14 @@ struct LoginForm: View {
     @Binding var username: String
     @Binding var password: String
     @Binding var homeserver: String
+    
+    let guessHomeserverURL: () -> Void
 
     var body: some View {
         VStack {
-            FormTextField(title: L10n.Login.Form.username, text: $username)
+            FormTextField(title: L10n.Login.Form.username, text: $username, onEditingChanged: { (editingChanged) in
```

Comment:
We're not using the argument anyways, might as well ignore it.
```suggestion
            FormTextField(title: L10n.Login.Form.username, text: $username, onEditingChanged: { _ in
```